plugins/uefi-capsule/meson.build

```diff
@@ -189,4 +189,24 @@ if get_option('tests')
   )
   test('uefi-self-test', e)
 endif
+
+# to use these do `sudo systemctl edit fwupd.service` and set
+# Environment="FWUPD_SYSFSFWDIR=/usr/share/installed-tests/fwupd"
```

Comment:
I think we want to add a test to gnome-desktop-tests to call out this thing too if the device was detected (IE this was done)